### PR TITLE
New version: Hashicorp.Nomad version 1.7.3

### PR DESCRIPTION
**Pull request by weimeng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135592)

### CODE REVIEW

**Comment by weimeng** [comment]
@microsoft-github-policy-service agree

File: manifests/h/Hashicorp/Nomad/1.7.3/Hashicorp.Nomad.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+PackageVersion: 1.7.3
+PackageLocale: en-US
+Publisher: HashiCorp
+PublisherUrl: https://www.hashicorp.com/
+PublisherSupportUrl: https://github.com/hashicorp/nomad/issues
+PrivacyUrl: https://www.hashicorp.com/privacy
+Author: David Lemler
+PackageName: Hashicorp Nomad
+PackageUrl: https://www.nomadproject.io/
+License: Mozilla Public License 2.0
```
Comment (mdanish-kh) [inline manifests/h/Hashicorp/Nomad/1.7.3/Hashicorp.Nomad.locale.en-US.yaml:14]:
```suggestion
License: Business Source License
```

https://github.com/hashicorp/nomad/commit/b3e30b1dfa185d9437a25830522da47b91f78816

File: manifests/h/Hashicorp/Nomad/1.7.3/Hashicorp.Nomad.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+LicenseUrl: https://github.com/hashicorp/nomad/blob/main/LICENSE
+Copyright: Copyright (c) 2015 HashiCorp, Inc.
```
Comment (mdanish-kh) [inline manifests/h/Hashicorp/Nomad/1.7.3/Hashicorp.Nomad.locale.en-US.yaml:16]:
```suggestion
Copyright: Copyright (c) 2023 HashiCorp, Inc.
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 642.)

**Pull request merged**